diff -Naur ../git.orig/m4/sqlite-threadsafe.m4 ./m4/sqlite-threadsafe.m4
--- ../git.orig/m4/sqlite-threadsafe.m4	2013-06-18 09:34:29.448822363 +0200
+++ ./m4/sqlite-threadsafe.m4	2013-06-18 09:32:54.412831527 +0200
@@ -45,7 +45,7 @@
   ]])],
        [ax_cv_sqlite_threadsafe=yes],
        [ax_cv_sqlite_threadsafe=no],
-       [ax_cv_sqlite_threadsafe=no])])
+       [ax_cv_sqlite_threadsafe=yes])])
 
   LIBS="$save_LIBS"
 ])
