--- git.orig/configure.ac	2013-06-25 13:07:27.393349931 +0200
+++ git/configure.ac	2013-06-25 12:48:11.760070337 +0200
@@ -684,42 +684,43 @@
 else
    have_tracker_fts="no  (disabled)"
 fi
+#
+#if test "x$have_tracker_fts" = "xyes"; then
+#   AC_DEFINE(HAVE_TRACKER_FTS, [1], [Define to 1 if tracker FTS is compiled])
+#
+#   OLD_CFLAGS="$CFLAGS"
+#   OLD_LIBS="$LIBS"
+#   CFLAGS="$SQLITE3_CFLAGS"
+#   LIBS="$SQLITE3_LIBS"
+#
+#   AC_MSG_CHECKING(whether SQLite3 has required FTS features)
+#   AC_RUN_IFELSE([AC_LANG_PROGRAM([[#include <sqlite3.h>]],
+#                                  [[sqlite3 *db;
+#                                    int rc;
+#                                    rc = sqlite3_open(":memory:", &db);
+#                                    if (rc!=SQLITE_OK) return -1;
+#                                    rc = sqlite3_exec(db, "create table a(text)", 0, 0, 0);
+#                                    if (rc!=SQLITE_OK) return -1;
+#                                    rc = sqlite3_exec(db, "create virtual table t using fts4(content='a',text)", 0, 0, 0);
+#                                    if (rc!=SQLITE_OK) return -1;]])],
+#                     [have_builtin_fts4=yes],
+#                     [have_builtin_fts4=no])
+#   AC_MSG_RESULT($have_builtin_fts4)
+#
+#   CFLAGS="$OLD_CFLAGS"
+#   LIBS="$OLD_LIBS"
+#
+#   if test "x$have_builtin_fts4" = "xyes" ; then
+#      AC_DEFINE(HAVE_BUILTIN_FTS, [], [Defined if Sqlite has FTS4 compiled in])
+#   fi
+#else
+AC_DEFINE(HAVE_TRACKER_FTS, [1], [Define to 0 if tracker FTS is not compiled])
+AC_DEFINE(HAVE_BUILTIN_FTS, [1], [Defined if Sqlite has FTS4 compiled in])
+#   $have_builtin_fts4="disabled"
+#fi
 
-if test "x$have_tracker_fts" = "xyes"; then
-   AC_DEFINE(HAVE_TRACKER_FTS, [1], [Define to 1 if tracker FTS is compiled])
-
-   OLD_CFLAGS="$CFLAGS"
-   OLD_LIBS="$LIBS"
-   CFLAGS="$SQLITE3_CFLAGS"
-   LIBS="$SQLITE3_LIBS"
-
-   AC_MSG_CHECKING(whether SQLite3 has required FTS features)
-   AC_RUN_IFELSE([AC_LANG_PROGRAM([[#include <sqlite3.h>]],
-                                  [[sqlite3 *db;
-                                    int rc;
-                                    rc = sqlite3_open(":memory:", &db);
-                                    if (rc!=SQLITE_OK) return -1;
-                                    rc = sqlite3_exec(db, "create table a(text)", 0, 0, 0);
-                                    if (rc!=SQLITE_OK) return -1;
-                                    rc = sqlite3_exec(db, "create virtual table t using fts4(content='a',text)", 0, 0, 0);
-                                    if (rc!=SQLITE_OK) return -1;]])],
-                     [have_builtin_fts4=yes],
-                     [have_builtin_fts4=no])
-   AC_MSG_RESULT($have_builtin_fts4)
-
-   CFLAGS="$OLD_CFLAGS"
-   LIBS="$OLD_LIBS"
-
-   if test "x$have_builtin_fts4" = "xyes" ; then
-      AC_DEFINE(HAVE_BUILTIN_FTS, [], [Defined if Sqlite has FTS4 compiled in])
-   fi
-else
-   AC_DEFINE(HAVE_TRACKER_FTS, [0], [Define to 0 if tracker FTS is not compiled])
-   $have_builtin_fts4="disabled"
-fi
-
-AM_CONDITIONAL(HAVE_BUILTIN_FTS, test "$have_builtin_fts4" = "yes")
-AM_CONDITIONAL(HAVE_TRACKER_FTS, test "$have_tracker_fts" = "yes")
+AM_CONDITIONAL(HAVE_BUILTIN_FTS, test "$have_builtin_fts4" != "aaa")
+AM_CONDITIONAL(HAVE_TRACKER_FTS, test "$have_tracker_fts" != "aaa")
 
 ####################################################################
 # Check for D-Bus requirements
