From 1376fa02b90d4f5cafa728483664be8785e461a4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonatan=20P=C3=A5lsson?= <jonatan.palsson@pelagicore.com>
Date: Fri, 22 Aug 2014 14:37:38 +0200
Subject: [PATCH] Add artist and AlbumArtURL to MediaContainer2

---
 libdleyna/server/props.c  | 31 +++++++++++++++++++++++++++++++
 libdleyna/server/server.c |  4 ++++
 2 files changed, 35 insertions(+)

diff --git a/libdleyna/server/props.c b/libdleyna/server/props.c
index fc26ec2..cda41ff 100644
--- a/libdleyna/server/props.c
+++ b/libdleyna/server/props.c
@@ -1505,6 +1505,23 @@ void dls_props_add_container(GVariantBuilder *item_vb,
 		prv_add_resources(item_vb, GUPNP_DIDL_LITE_OBJECT(object),
 				  filter_mask, FALSE);
 
+
+	if (filter_mask & DLS_UPNP_MASK_PROP_ARTIST) {
+		GUPnPDIDLLiteObject *obj = GUPNP_DIDL_LITE_OBJECT(object);
+		const char *artist = gupnp_didl_lite_object_get_artist(obj);
+		prv_add_string_prop (item_vb,
+				     DLS_INTERFACE_PROP_ARTIST,
+				     artist);
+	}
+
+	if (filter_mask & DLS_UPNP_MASK_PROP_ALBUM_ART_URL) {
+		GUPnPDIDLLiteObject *obj = GUPNP_DIDL_LITE_OBJECT(object);
+		const char *url = gupnp_didl_lite_object_get_album_art(obj);
+		prv_add_string_prop (item_vb,
+				     DLS_INTERFACE_PROP_ARTIST,
+				     url);
+	}
+
 	res = prv_get_matching_resource(GUPNP_DIDL_LITE_OBJECT(object),
 					protocol_info);
 	if (res) {
@@ -2039,6 +2056,20 @@ GVariant *dls_props_get_container_prop(const gchar *prop,
 		DLEYNA_LOG_DEBUG("Prop %s = %u", prop, uint_val);
 
 		retval = g_variant_ref_sink(g_variant_new_uint32(uint_val));
+	} else if (!strcmp(prop, DLS_INTERFACE_PROP_ARTIST)) {
+		const char *strval = gupnp_didl_lite_object_get_artist(
+						GUPNP_DIDL_LITE_OBJECT(object));
+
+		DLEYNA_LOG_DEBUG("Prop %s = %u", prop, uint_val);
+
+		retval = g_variant_ref_sink(g_variant_new_string(strval));
+	} else if (!strcmp(prop, DLS_INTERFACE_PROP_ALBUM_ART_URL)) {
+		const char *strval = gupnp_didl_lite_object_get_album_art(
+						GUPNP_DIDL_LITE_OBJECT(object));
+
+		DLEYNA_LOG_DEBUG("Prop %s = %u", prop, uint_val);
+
+		retval = g_variant_ref_sink(g_variant_new_string(strval));
 	} else if (!strcmp(prop, DLS_INTERFACE_PROP_RESOURCES)) {
 		retval = g_variant_ref_sink(
 			prv_compute_resources(object, DLS_UPNP_MASK_ALL_PROPS,
diff --git a/libdleyna/server/server.c b/libdleyna/server/server.c
index 425f764..ec725de 100644
--- a/libdleyna/server/server.c
+++ b/libdleyna/server/server.c
@@ -341,6 +341,10 @@ static const gchar g_server_introspection[] =
 	"       access='read'/>"
 	"    <property type='x' name='"DLS_INTERFACE_PROP_SIZE"'"
 	"       access='read'/>"
+	"    <property type='s' name='"DLS_INTERFACE_PROP_ARTIST"'"
+	"       access='read'/>"
+	"    <property type='s' name='"DLS_INTERFACE_PROP_ALBUM_ART_URL"'"
+	"       access='read'/>"
 	"  </interface>"
 	"  <interface name='"DLS_INTERFACE_MEDIA_ITEM"'>"
 	"    <method name='"DLS_INTERFACE_GET_COMPATIBLE_RESOURCE"'>"
-- 
2.0.0

