From 059047dfcabeea25df0ef69ef62efaf5c39c3e40 Mon Sep 17 00:00:00 2001
From: Maciej Borzecki <maciej.borzecki@open-rnd.pl>
Date: Mon, 5 May 2014 15:54:24 +0200
Subject: [PATCH 1/2] systemd: add lresolv to build with eglibc

---
 Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index c9eb79e1704065a5343e88611a81268c3ecb76d6..898ffa58ed5efc2762ef5d61122fd6ed8d517b26 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2533,7 +2533,8 @@ libudev_la_LDFLAGS = \
 
 libudev_la_LIBADD = \
 	libsystemd-internal.la \
-	libsystemd-shared.la
+	libsystemd-shared.la \
+	-lresolv
 
 pkgconfiglib_DATA += \
 	src/libudev/libudev.pc
@@ -4297,6 +4298,7 @@ pam_systemd_la_LIBADD = \
 	libsystemd-capability.la \
 	libsystemd-internal.la \
 	libsystemd-shared.la \
+	-lresolv \
 	$(PAM_LIBS)
 
 pamlib_LTLIBRARIES = \
-- 
1.9.0

