diff -ur qtwebkit-opensource-src-5.1.1/Source/ThirdParty/ANGLE/src/compiler/glslang.y qtwebkit-opensource-src-5.1.1.fixed/Source/ThirdParty/ANGLE/src/compiler/glslang.y
--- qtwebkit-opensource-src-5.1.1/Source/ThirdParty/ANGLE/src/compiler/glslang.y	2013-08-25 20:04:43.000000000 +0200
+++ qtwebkit-opensource-src-5.1.1.fixed/Source/ThirdParty/ANGLE/src/compiler/glslang.y	2014-04-02 15:25:54.436352726 +0200
@@ -47,6 +47,7 @@
 %expect 1 /* One shift reduce conflict because of if | else */
 %pure-parser
 %parse-param {TParseContext* context}
+%lex-param {YYLEX_PARAM}
 
 %union {
     struct {

diff -ur qtwebkit-opensource-src-5.1.1/Source/WebKit/qt/WidgetApi/qwebviewaccessible_p.h qtwebkit-opensource-src-5.1.1.fixed/Source/WebKit/qt/WidgetApi/qwebviewaccessible_p.h
--- qtwebkit-opensource-src-5.1.1/Source/WebKit/qt/WidgetApi/qwebviewaccessible_p.h	2013-08-25 20:04:54.000000000 +0200
+++ qtwebkit-opensource-src-5.1.1.fixed/Source/WebKit/qt/WidgetApi/qwebviewaccessible_p.h	2014-04-02 17:30:08.228405250 +0200
@@ -22,7 +22,7 @@
 
 #include <qaccessible.h>
 #include <qaccessibleobject.h>
-#if QT_VERSION >= QT_VERSION_CHECK(5, 1, 0)
+#if QT_VERSION >= QT_VERSION_CHECK(5, 1, 0) && QT_VERSION < QT_VERSION_CHECK(5, 2, 0)
 #include <private/qaccessiblewidget_p.h>
 #else
 #include <qaccessiblewidget.h>
