diff -ur systemd-199.orig/rules/60-persistent-storage.rules systemd-199/rules/60-persistent-storage.rules
--- systemd-199.orig/rules/60-persistent-storage.rules	2013-04-03 15:17:39.455333262 +0200
+++ systemd-199/rules/60-persistent-storage.rules	2013-04-03 15:18:24.931338988 +0200
@@ -67,7 +67,7 @@
   IMPORT{builtin}="blkid --noraid"
 
 # probe filesystem metadata of disks
-KERNEL!="sr*", IMPORT{builtin}="blkid"
+KERNEL!="sr*", IMPORT{program}="/sbin/blkid -o udev -p $tempnode"
 
 # watch metadata changes by tools closing the device after writing
 KERNEL!="sr*", OPTIONS+="watch"
