diff --git a/rygel-lms-database.vala b/rygel-lms-database.vala
index 116d40f..f272189 100644
--- a/rygel-lms-database.vala
+++ b/rygel-lms-database.vala
@@ -107,7 +107,7 @@ public class Rygel.LMS.Database {
         if (this.db.errcode () != Sqlite.OK) {
             throw new DatabaseError.OPEN ("Failed to open '%s': %d",
                                           db_path,
-                                          this.db.errcode);
+                                          (int) this.db.errcode);
         }
 
         this.db.create_function ("contains",
