From 517bcd8a1edf43c68ba898019fb6688655c0feab Mon Sep 17 00:00:00 2001
From: Joakim Gross <joakim.gross@pelagicore.com>
Date: Tue, 8 Mar 2016 15:36:31 +0100
Subject: [PATCH] Adds correct DBus libraries path if installed outside of
 standard paths

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2750c4e..d252101 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -152,6 +152,10 @@ if ("${USE_INSTALLED_DBUS}" STREQUAL "OFF")
     link_directories(
         ${DBus_INCLUDE_DIRS}/dbus/.libs
     )
+else ()
+    link_directories(
+        ${DBus_LIBRARY_DIRS}
+    )
 endif()
 
 # DBus source files
-- 
2.7.0

