From 8a83dbb456cf0cf057e546bf9e8bcb9fbbf37bdf Mon Sep 17 00:00:00 2001
From: Joakim Gross <joakim.gross@pelagicore.com>
Date: Tue, 8 Mar 2016 15:33:17 +0100
Subject: [PATCH] Adds required dependencies to pc file

---
 CommonAPI-DBus.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CommonAPI-DBus.pc.in b/CommonAPI-DBus.pc.in
index e93ebee..894e0b5 100644
--- a/CommonAPI-DBus.pc.in
+++ b/CommonAPI-DBus.pc.in
@@ -6,6 +6,7 @@ includedir=@CMAKE_INSTALL_PREFIX@/include
 Name: @PROJECT@
 Description: GENIVI Common API D-Bus C++ Library
 Version: @COMPONENT_VERSION@
+Requires: CommonAPI dbus-1
 Libs: -L${libdir} -lCommonAPI-DBus
 Cflags: -I${includedir}/CommonAPI-@COMMONAPI_API_HEADER_VERSION@
 
-- 
2.7.0

