From 772eb8c08d8449d80b8afd8ae3872d8bd1f30120 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Erik=20Bot=C3=B6?= <erik.boto@pelagicore.com>
Date: Fri, 5 Sep 2014 15:32:21 +0200
Subject: [PATCH 2/2] Disable doc (fails to build)

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 746c750..b32da3d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = src data tests po doc examples
+SUBDIRS = src data tests po examples
 
 pkgconfig_DATA= rygel-core-2.4.pc rygel-renderer-2.4.pc rygel-server-2.4.pc
 pkgconfigdir = $(libdir)/pkgconfig
-- 
2.1.0

