diff -uNr rygel-0.21.6.orig/data/rygel.conf rygel-0.21.6/data/rygel.conf
--- rygel-0.21.6.orig/data/rygel.conf	2014-07-11 15:48:50.575153310 +0200
+++ rygel-0.21.6/data/rygel.conf	2014-07-11 16:02:07.947158929 +0200
@@ -24,7 +24,7 @@
 
 # Default media engine to load. If not specified, the engine directory is
 # searched recursively and the first engine found is loaded.
-media-engine=librygel-media-engine-gst.so
+# media-engine=librygel-media-engine-gst.so
 
 # List of network interfaces to attach rygel to. You can also use network IP or
 # even ESSID for wireless networks on Linux. Leave it blank for dynamic

